metricbeat/mb/testing/data/config.yml

```diff
@@ -1,7 +0,0 @@
-type: http
```

Comment:
We don't need this anymore?

Reply:
I couldn't find where this was being used, so I removed it.

Could this be the cause of the change in apache hostname?

Reply:
No. It was there as reference. Please, undo :slightly_smiling_face:

Reply:
Apache hostname is changing because it takes the hostname of the beat and writes it in each execution at the metricset level. We discussed about leaving it like this for compatibility reasons. [Here](https://github.com/elastic/beats/blob/master/metricbeat/module/apache/status/status.go#L89) you can see that `m.Host` is passed to the `eventMapping` which sets it [here](https://github.com/elastic/beats/blob/master/metricbeat/module/apache/status/data.go#L180)

Reply:
Wdyt about [adding it as example in the godocs](9b3fc2355e2599be1d5c1a1623830620d36895e1)? If my future self finds this file alone here he will probably wonder again what it is.

Reply:
> Apache hostname is changing because it takes the hostname of the beat and writes it in each execution at the metricset level. We discussed about leaving it like this for compatibility reasons. [Here](https://github.com/elastic/beats/blob/master/metricbeat/module/apache/status/status.go#L89) you can see that `m.Host` is passed to the `eventMapping` which sets it [here](https://github.com/elastic/beats/blob/master/metricbeat/module/apache/status/data.go#L180)

Ok, I see it also changes when running `TestAll` with generate, I won't change this in this PR.